app/controllers/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
+  def create
+    auth_hash = request.env['omniauth.auth']
+    google_id = auth_hash['uid']
+    email = auth_hash['info']['email']
+    token = auth_hash['credentials']['token']
+
+    user = User.find_or_create_by(email: email)
+    user.update(google_id: google_id)
+
```

Comment:
should we add a `session[:token] = token` here? if we want to do something else with the google oauth later?

Reply:
makes sense to me